app.yml

```diff
@@ -0,0 +1,7 @@
+default_events:
+  - pull_request
+
+default_permissions:
+  pull_requests: read
+  metadata: read
```

Comment:
Not sure if this will be necessary. When we run the Github App we will see.

Reply:
it's required by default. I think we get it even if we don't set it here